Detectors/EMCAL/simulation/src/Digitizer.cxx

```diff
@@ -35,6 +37,39 @@ void Digitizer::init()
 {
   mSimParam = SimParam::GetInstance();
   mRandomGenerator = new TRandom3(std::chrono::high_resolution_clock::now().time_since_epoch().count());
+
+  float tau = mSimParam->GetTimeResponseTau();
+  float N = mSimParam->GetTimeResponsePower();
+  mTimeBinOffset = ((int)tau + 0.5);
+  mSignalFractionInTimeBins.clear();
+  TF1 RawResponse("RawResponse", rawResponseFunction, 0, 256, 5);
+  RawResponse.SetParameters(1., 0., tau, N, 0.);
+  double RawResponseTotalIntegral = (N > 0) ? pow(N, -N) * exp(N) * TMath::Gamma(N) : 1.;
+  for (int j = 0; j < 1000; j++) {
```

Comment:
Prefer not to hardcode `1000`. I guess there must be some constant defining the number of bins.

Reply:
Indeed, this should be the number of time samples, and to be defined in EMCAL/base/include/EMCALBase/Constants.h